client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -73,11 +74,17 @@ export const EquipmentDetailView = () => {
 
   useEffect(() => {
     if (!data) return;
+    let properties = JSON.parse(data.properties);
```

Comment:
Should we merge the asset catalogue properties with the asset properties here? Or should we do it in the graphql layer?